mltsp/Flask/flask_app.py

```diff
@@ -123,7 +103,7 @@
 logging.basicConfig(filename=cfg.ERR_LOG_PATH, level=logging.WARNING)
 
 # RethinkDB config:
-RDB_HOST =  os.environ.get('RDB_HOST') or 'localhost'
+RDB_HOST = os.environ.get('RDB_HOST') or 'localhost'
```

Comment:
We never set them, just adopting this from the RDB examples so that others may easily set them if they wish. But yes, something in the README could be helpful.